Modules/rat_command.py

```
@@ -85,10 +85,14 @@ class Commands:
             # someone didn't set me.
             raise CommandException(f"Bot client has not been created or not handed to Commands.")
 
+            # command didn't start with prefix, ignore it
+        if not message.startswith(cls.prefix):
+            cls.log.debug(f"caught a command without prefix: {message}")
+            return None
```

This should probably raise an exception, rather than a null return.